src/Tribe/RSVP.php

```diff
@@ -2061,6 +2061,11 @@ public function get_attendee( $attendee, $post_id = 0 ) {
 
 			// Attendee Meta
 			'attendee_meta' => $meta,
+
+			// Handle initial Attendee flags.
+			// @todo Make these live in future IAC work.
+			'is_subscribed' => false,
+			'is_purchaser'  => true,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 7 space(s) between "'is_purchaser'" and double arrow, but found 2.

<sub>hash: a941c82a89e26b48fcf792046ee1e879</sub>